docs/source/tutorials/GEDI/GEDI_L3_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"collection name: {collection_name} | file_name: {file_name}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "332fe4ce",
+   "metadata": {},
+   "source": [
+    "**Download file from ORNL DAAC S3**\n",
+    "\n",
+    "To download the file from the source, temporary s3 credentials are required for `maap` package. You can explicitly request `s3_cred_endpoint` for the credentials or use the code below to get credentials and download the file in your workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "47108597",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_s3_creds(url):\n",
+    "    return maap.aws.earthdata_s3_credentials(url)\n",
+    "\n",
+    "def get_s3_client(s3_cred_endpoint):\n",
+    "    creds=get_s3_creds(s3_cred_endpoint)\n",
+    "    boto3_session = boto3.Session(\n",
+    "            aws_access_key_id=creds['accessKeyId'],\n",
+    "            aws_secret_access_key=creds['secretAccessKey'],\n",
+    "            aws_session_token=creds['sessionToken']\n",
+    "    )\n",
+    "    return boto3_session.client(\"s3\")\n",
+    "\n",
+    "def download_s3_file(s3, bucket, collection_name, file_name):\n",
+    "    os.makedirs(\"/projects/gedi_l3\", exist_ok=True) # create directories, as necessary\n",
+    "    download_path=f\"/projects/gedi_l3/{file_name}.tif\"\n",
+    "    s3.download_file(bucket, f\"gedi/{collection_name}/data/{file_name}.tif\", download_path)\n",
+    "    return download_path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "4d9a603c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_cred_endpoint= 'https://data.ornldaac.earthdata.nasa.gov/s3credentials' \n",
+    "s3=get_s3_client(s3_cred_endpoint)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "f1fabdcc",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'/projects/gedi_l3/GEDI03_elev_lowestmode_stddev_2019108_2020287_002_02.tif'"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bucket=\"ornl-cumulus-prod-protected\"\n",
+    "download_path=download_s3_file(s3, bucket, collection_name, file_name)\n",
+    "download_path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b8a2de6a",
+   "metadata": {},
+   "source": [
+    "Now, we have the file in our local workspace. It's time to visualize it using `rasterio` package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa07ffbe",
+   "metadata": {},
+   "source": [
+    "**[Optional] Visualization using rasterio:**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44498ee3",
+   "metadata": {},
+   "source": [
+    "The downloaded file is too big to read and visualize directly so we might need to scale it down and view it as a small thumbinal. "
```

Comment:
```suggestion
    "The downloaded file is too big to read and visualize directly so we might need to scale it down and view it as a small thumbnail. "
```
Combine text cell with the one above, just add a newline after the title

Reply:
Nice catch